src/main/java/sirius/biz/tenants/packages/Packages.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.biz.tenants.packages;
+
+import com.typesafe.config.Config;
+import sirius.kernel.Sirius;
+import sirius.kernel.di.std.Register;
+import sirius.kernel.nls.NLS;
+import sirius.kernel.settings.Extension;
+import sirius.web.security.UserContext;
+
+import java.util.List;
+
+/**
+ * This helper class is a wrapper for the <tt>security.packages</tt> configuratuions.
+ */
+@Register(classes = Packages.class)
+public class Packages {
```

Comment:
how do you calculate the permissions from the packages and upgrades?

Reply:
that comes in one of the next PRs right? The packages and upgrades need to be defined in the security profiles and will get extrapolated to all applicable permissions